src/ape_node/provider.py

```diff
@@ -38,6 +39,47 @@
     EthereumNodeProvider,
 )
 
+Alloc = dict[str, dict[str, Any]]
+
+
+def create_genesis_data(alloc: Alloc, chain_id: int) -> GenesisDataTypedDict:
+    """
+    A wrapper around genesis data for py-geth that
+    fills in more defaults.
+    """
+    return {
+        "alloc": alloc,
+        "config": {
+            "arrowGlacierBlock": 0,
+            "berlinBlock": 0,
+            "byzantiumBlock": 0,
+            "cancunTime": 0,
+            "chainId": chain_id,
```

Comment:
i had to fix this locally to get it to work: https://github.com/ethereum/py-geth/issues/220